Finishlinelab3/finishlinelab3infra.yaml

```diff
@@ -24,57 +28,50 @@ Parameters:
     Default: 0.0.0.0/0
     AllowedPattern: '(\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})/(\d{1,2})'
     ConstraintDescription: must be a valid IP CIDR range of the form x.x.x.x/x.
-Mappings:
-  AWSInstanceType2Arch:
-    t2.micro:
-      Arch: HVM64
-  AWSInstanceType2NATArch:
-    t2.micro:
-      Arch: NATHVM64
-  AWSRegionArch2AMI:
-    us-east-1:
-      HVM64: ami-0080e4c5bc078760e
-      HVMG2: ami-0aeb704d503081ea6
 Resources:
-  EC2Instance:
+  BaninaWebServer:
     Type: 'AWS::EC2::Instance'
     Properties:
       InstanceType: !Ref InstanceType
       SecurityGroups:
         - !Ref InstanceSecurityGroup
       KeyName: !Ref KeyName
-      ImageId: !FindInMap 
-        - AWSRegionArch2AMI
-        - !Ref 'AWS::Region'
-        - !FindInMap 
-          - AWSInstanceType2Arch
-          - !Ref InstanceType
-          - Arch
-  InstanceSecurityGroup:
-    Type: 'AWS::EC2::SecurityGroup'
-    Properties:
-      GroupDescription: Enable SSH access via port 22
-      SecurityGroupIngress:
-        - IpProtocol: tcp
-          FromPort: '22'
-          ToPort: '22'
-          CidrIp: !Ref SSHLocation
+      ImageId: !Ref AmiId 
+      Tags:
+        - Key: Name
+          Value: BaninaWebServer
+  BaninaWebServerSecurityGroup:
+  Type: AWS::EC2::SecurityGroup
+  Properties:
+    GroupDescription: Enable HTTP access via port 80 and SSH access
+    SecurityGroupIngress:
+    - IpProtocol: tcp
+      FromPort: '80'
+      ToPort: '80'
+      CidrIp: 0.0.0.0/0
+    - IpProtocol: tcp
+      FromPort: '22'
+      ToPort: '22'
+      CidrIp: 0.0.0.0/0          
+  BaninaDBSecurityGroup:
+  Type: AWS::RDS::DBSecurityGroup
+  Properties:
+    DBSecurityGroupIngress:
+      EC2SecurityGroupName:
+        Ref: WebServerSecurityGroup
+    GroupDescription: Frontend Access        
 Outputs:
   InstanceId:
     Description: InstanceId of the newly created EC2 instance
-    Value: !Ref EC2Instance
+    Value: !Ref BaninaWebserver
   AZ:
     Description: Availability Zone of the newly created EC2 instance
     Value: !GetAtt 
-      - EC2Instance
+      - BaninaWebserver
       - AvailabilityZone
   PublicDNS:
     Description: Public DNSName of the newly created EC2 instance
     Value: !GetAtt 
-      - EC2Instance
+      - BaninaWebserver
       - PublicDnsName
-  PublicIP:
-    Description: Public IP address of the newly created EC2 instance
-    Value: !GetAtt 
-      - EC2Instance
-      - PublicIp 
```

Comment:
Please us this in place of PublicDNS. Work with Corker and Charity

Reply:
Done